docs/dependencies/introduction.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Package types
+---
+
+Previously you learned how to [start using Eik in an application](/docs/introduction/workflow), including how to do import mapping so shared dependencies only get downloaded, parsed and run once. For this to work, though, there must be shared dependencies available on the Eik server.
+
+In the Managing dependencies section you'll learn how to:
+
+- publish shared dependencies to the `npm` namespace on your Eik server
+- how to create and publish import maps to your Eik server
+- how to create aliases for both of the above
+
+## Eik package types
+
+It's worth mentioning at this stage that Eik differentiates between three types of packages:
+
+- `npm`
+- `map`
+- `package`
+
+You'll see this distinction in [`eik.json`](/docs/reference/eik-json/) and in the URL where your package gets published. Each package type has its own namespace. This is to avoid accidental naming collisions between, say, the `npm` module `lit` and the import map `lit`.
```

Comment:
The difference between the package types isn't really explained in the `eik.json` documentation. It mentions it's to avoid collisions, which makes sense for `map` and `npm`. 
However there is no explanation for `package` being a type. Given that it's a bit of a strange name I think we should mention explain here exactly what the intention with especially `package` is.

Reply:
It's really only to avoid collision, as far as I can tell. It's explained it's intended for application code in the paragraph below.

> Application code should be published to the `package` namespace. This is also the default, if no other type is configured in `eik.json`.